### PR TITLE
Feature/maven

### PR DESCRIPTION
**Pull request by hienng99**

### CODE REVIEW

File: src/main/java/com/company/Test.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.company;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class Test {
+
+	public static void main(String[] args) {
+
+		boolean result = StringUtils.isNumeric("Hien");
+		System.out.println(result ? "Day la so.":"Day khong phai la so.");
```
Comment (QuangCuongIT) [inline src/main/java/com/company/Test.java]: Our customer is in the EU. Please never use Vietnamese in the code :)

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  		</plugin>
+  	</plugins>
+  </build>
+  
+  <dependencies>
+   	<!-- https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 -->
+	<dependency>
+	    <groupId>org.apache.commons</groupId>
+	    <artifactId>commons-lang3</artifactId>
+	    <version>3.12.0</version>
```
Comment (QuangCuongIT) [inline pom.xml]: What is the scope of this dependency? If it's the default scope, what is it? And why do you use it?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.company</groupId>
+  <artifactId>hello-maven</artifactId>
+  <version>1.0</version>
```
Comment (QuangCuongIT) [inline pom.xml]: use 1.0.0 for version, It's 
image

Comment (hienng99) [reply pom.xml]: I have adjusted

File: pom.xml (hunk 2 above)
Comment (hienng99) [reply pom.xml]: It is compile scope, so the dependency is always appear in classpath.

File: src/main/java/com/company/Test.java (hunk 1 above)
Comment (hienng99) [reply src/main/java/com/company/Test.java]: I have adjusted

**Review by QuangCuongIT — APPROVED** [review]

**Pull request merged**